_minutes/2022-09-29-wecg.md

```diff
@@ -0,0 +1,167 @@
+ * [jonathan] That would require dynamic scripts (strings).
+ * [rob] For scripting.executeScript we could consider supporting the existing args key for file (not just func), which would execute the file as code in a function scope with the arguments being passed, and similarly for the scripting.registerContentScripts API. While conceptually understandable to JS devs, it may be more complicated to implement by browsers though.
+ * [rob] While not a blocker, it would be nice to consider this functionality in the context of the user script API ([#279](https://github.com/w3c/webextensions/issues/279)).
+
+[Issue 285](https://github.com/w3c/webextensions/issues/285): Inconsistency: CSP sandbox directive and content script and style injection into pages
+
+ * [rob] I noticed this issue in the past and discussed it with Devlin (Chrome), and designed an update to the content script API to address this inconsistency: https://bugzilla.mozilla.org/show_bug.cgi?id=1411641#c43. Timothy and I should probably look into this more closely and revisit the issue next meeting.
+
+[Issue 205](https://github.com/w3c/webextensions/issues/205): Label naming for issues where consensus has been achieved
+
+ * [carlos] We've introduced some labels, so maybe this is no longer needed?
+ * [rob] This has been addressed with the supportive/implemented labels.
+ * [timothy] Typically, when there are two or more implementations, there is consensus.
+
+[Issue 274](https://github.com/w3c/webextensions/issues/274): Proposal: i18n.getLanguageDictionary
+
+ * [carlos] Discussion on how to split this API from the bigger issue [#258](https://github.com/w3c/webextensions/issues/258), opinions on naming?
+ * [simeon] Not concerned with the naming specifically; last time I was confused about the specific word “dictionary” in the signature, but don't want to focus too much on naming for the moment. I'm supportive of this in abstract.
+ * [timothy] (Apple/Safari: supportive) I'm supportive of it for Safari and have already added the `supportive: safari` label.
+ * [tomislav] (Mozilla/Firefox: neutral) Not opposed either.
+
+[Issue 258](https://github.com/w3c/webextensions/issues/258): Per-extension language preferences
+
+ * [carlos] This is the other half of the previously discussed issue.
+ * [tomislav] Would we need to provide a UI, or is an extension API sufficient?
+ * [carlos] The value of this API is that both the browser and user/extension can change this.
+ * [tomislav] Then I don't understand why this is per extension.
+ * [carlos] The browser would allow a native toggle per extension language, like Android.
+ * [tomislav] I will read the full issue later. Note that implementing UI is a high bar, an API-only approach would be preferred.
+ * [oliver] Would this also affect things like the extension name?
+ * [timothy] Curious whether there is user demand for this feature. Is this pretty common for bilingual people to change this?
```

Comment:
@xeenon This feature is very common on the Web (not just for extensions). For example:
In [Gmail settings](https://mail.google.com/mail/u/0/#settings/general), users can change Gmail display language. Another example see this [video](https://www.youtube.com/watch?v=DUKnNWwcNvo).

Reply:
Please use the specific issues to discuss the topic. The meeting notes are only used to record discussions that have happened during the meeting.